### PR TITLE
Created stacked area chart component

### PR DESCRIPTION
**Pull request by kevinylee1991**
Looks like below:

Screen Shot 2020-02-10 at 7 42 40 PM

Todos are to create a legend for it and to fit it into the prometheus alarms page.

### CODE REVIEW

File: src/app/core/themes/defaultTheme.js [hunk 1]
```diff
@@ -214,6 +214,11 @@ const theme = {
       aws: '#FF9800',
       azure: '#4AA3DF',
     },
+    stackedAreaChart: {
```
Comment (jruddell) [inline src/app/core/themes/defaultTheme.js]: I'd like to stop making custom theme colors for individual items. Its better if we point at existing types of colors. `palette.succes`, `palette.error`, `palette.warning`

File: src/app/core/components/graphs/StackedAreaChart.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  name: string
+  color: string
+}
+
+// types should be a list of strings that also show up in AreaChartEntry as a property
+interface Props {
+  data: AreaChartEntry[]
+  width: number
+  height: number
+  xAxis: string
```
Comment (jruddell) [inline src/app/core/components/graphs/StackedAreaChart.tsx]: is there a required entry in `data` that has to match the `xAxis` prop?

File: src/app/core/components/graphs/PieGraph.tsx [hunk 3]
```diff
@@ -45,7 +46,7 @@ const PieGraph = ({
         {...rest}
       >
         {items.map((entry, index) => (
-          <Cell key={entry.name} fill={theme.palette.pieChart[entry.color]} />
+          <Cell key={entry.name} fill={path(entry.color.split('.'), theme.palette)} />
```
Comment (brennancheung) [inline src/app/core/components/graphs/PieGraph.tsx]: We have a utility to save yourself needing to do the `split`.

`import { pathStr } from 'utils/fp'`

File: src/app/core/themes/defaultTheme.js [hunk 4]
```diff
@@ -128,6 +128,7 @@ const theme = {
       contrastText: '#FFF',
     },
     warning: {
+      lighter: '#FED16C',
```
Comment (jruddell) [inline src/app/core/themes/defaultTheme.js]: is `light` too dark so we need this? it would be nice to keep things consistent if we can

**Pull request merged**